### ISSUE TITLE
Error while loading another XDL file

### ISSUE THREAD

**Issue by vanilla-extracts** [smehrbrodt/libreoffice-starter-extension#9]
Sorry this does not _strictly_ talk about this project.
The code for my extension is available [here](https://github.com/coco33920/wakatime-libreofficel)
The Test.xdl file works fine with this project i cloned, 
The debugs shows the triggered function is called and the switch statements is working okay.

Also if i put a line like this,
```java
    DialogHelper.showErrorMessage(m_xContext, null, "test : " + action);
```
_before_ the Test object declation it _shows_
but if i put it _after_ it do not shows, however it works well in this project cloned so my guess it was about consuming the context seems false?

I'm sorry for putting this there, but finding documentation and community for this kind of development in Java is like _really_ difficult.

**Comment by smehrbrodt** [comment]
> The code for my extension is available [here](https://github.com/coco33920/wakatime-libreofficel)

That link does not work for me.

**Comment by vanilla-extracts** [comment]
Yeah sorry broken link, [here is one](https://github.com/coco33920/wakatime-libreoffice)

**Comment by smehrbrodt** [comment]
Ok, so have you tried debugging the Java side?

**Comment by vanilla-extracts** [comment]
Yes the switch statements is called, as explained if I try to display a message box it kind of works?.

The XDL loading class is valid because i tried it with this project and it works fine.
I do not understand this strange behaviour I must have done something wrong but I can't pin it down
